clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/configuration/KubernetesAccountConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.security.configuration;
+
+import com.netflix.spinnaker.clouddriver.kubernetes.security.KubernetesNamedAccountCredentials;
+import com.netflix.spinnaker.clouddriver.security.ProviderVersion;
+
+public interface KubernetesAccountConfiguration {
+  boolean handles(String configurationName, ProviderVersion providerVersion);
+  void configure(KubernetesNamedAccountCredentials credentials);
```

Comment:
Seems that all that `configure` does it add a namer in the only implementation - is that the intent? Or will more configuration be added in the future? As it stands I would name this differently, and maybe even structure it differently (I don't see why this needs an interface)